app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -544,6 +364,48 @@ class TopicFragmentTest {
     return launch(createTopicActivityIntent(internalProfileId, topicId))
   }
 
+  private fun clickTabAtPosition(position: Int) {
+    onView(
+      Matchers.allOf(
```

Comment:
Prefer directly importing allOf() for consistency.

Reply:
ah, missed it. Done